content/sensu-go/5.2/api/auth.md

```diff
@@ -34,3 +41,168 @@ HTTP/1.1 200 OK
 description          | Tests a given username and password.
 example url          | http://hostname:8080/api/core/v2/auth/test
 response codes       | <ul><li>**Valid credentials**: 200 (OK)</li><li> **Invalid credentials**: 401 (Unauthorized)</li><li>**Error**: 500 (Internal Server Error)</li></ul>
+
+## The `/providers` API endpoints {#the-providers-api-endpoints}
+
+**ENTERPRISE ONLY**: Authentication providers in Sensu Go require a Sensu Enterprise license. To activate your Sensu Enterprise license, see the [getting started guide][2].
+
+### `/providers` (GET) {#providers-get}
+
+The `/providers` API endpoint provides HTTP GET access to authentication provider configuration in Sensu.
+
+#### API Specification {#providers-get-specification}
+
+/providers (GET)  | 
+---------------|------
+description    | Returns the list of active authentication providers.
+example url    | http://hostname:8080/api/enterprise/v2/providers
+response type  | Array
+response codes | <ul><li>**Success**: 200 (OK)</li><li>**Error**: 500 (Internal Server Error)</li></ul>
+output         | {{< highlight json >}}
+[
+  {
+    "servers": [
+      {
+        "host": "127.0.0.1",
+        "binding": {
+          "user_dn": "cn=binder,dc=acme,dc=org",
+          "password": "P@ssw0rd!"
+        },
+        "group_search": {
+          "base_dn": "dc=acme,dc=org"
+        },
+        "user_search": {
+          "base_dn": "dc=acme,dc=org"
+        }
+      }
+    ],
+    "metadata": {
+      "name": "default"
+    }
+  }
+]
+{{< /highlight >}}
+
+### `/providers` (POST) {#providers-post}
+
+The `/providers` API endpoint provides HTTP POST access to apply an [authentication provider][1] configuration to Sensu.
+
+#### API Specification {#providers-post-specification}
+
+/providers (POST) | 
```

Comment:
TODO(palourde): Validate if the POST endpoint is `/providers` or `/providers/:type`; I suspect it's the latter but I need to double check